### PR TITLE
Add Shape Area safety button

### PR DESCRIPTION
**Pull request by plscks**

### CODE REVIEW

File: libconglomerate.user.js [hunk 1]
```diff
@@ -1928,6 +1929,7 @@ function libSettings() {
         ['safety', 'b', 'Safe Well', 'well', 'Adds a safety to creating an Arcane Well on Conduit.'],
         ['safety', 'b', 'Safe Mark', 'mark', 'Adds a safety to creating a Nexal Mark on Conduit.'],
         ['safety', 'b', 'Safe Heal Self', 'hself', 'Adds a safety to Heal Self on Sorcerer.'],
+        ['safety', 'b', 'Safe Shape Area', 'shape', 'Adds a safety to Shape Area on Conduit.'],
```
Comment (AnneTrue) [inline libconglomerate.user.js]:
```suggestion
        ['safety', 'b', 'Safe Shape Area', 'shape', 'Adds a safety to Shape Area for Sorcerer exits with Deep Spellcraft.'],
```

File: libconglomerate.user.js [hunk 2]
```diff
@@ -576,6 +576,7 @@ function safebuttons() {
     if (getSetting('safety-extra-well') == 'true') { safetyButtons("//form[@name='skilluse']/input[@type='submit' and starts-with(@value, 'Open Arcane Well')]", 'disable'); }
     if (getSetting('safety-extra-mark') == 'true') { safetyButtons("//form[@name='skilluse']/input[@type='submit' and starts-with(@value, 'Create Nexal Mark')]", 'disable'); }
     if (getSetting('safety-extra-hself') == 'true') { safetyButtons("//form[@name='skilluse']/input[@type='submit' and starts-with(@value, 'Heal Self')]", 'disable'); }
+    if (getSetting('safety-extra-shape') == 'true') { safetyButtons("//form[@name='spellaaoettack']/input[@type='submit' and @value='Shape Area Spell (5 AP)']", 'disable'); }
```
Comment (AnneTrue) [inline libconglomerate.user.js]: I recommend using the `starts-with` directive and removing the `(5 AP)` cost string, as with the other safeties.
Doing so will make the code less brittle (i.e. not breaking if the developers change the cost of the ability)

**Review by AnneTrue — CHANGES_REQUESTED** [review]
Thanks for the MR! However there's two nitpicks before I can merge.

**Comment by plscks** [comment]
I've made the requested changes, thank you for the input it is appreciated.

**Review by AnneTrue — APPROVED** [review]

**Pull request merged**